app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -123,6 +123,23 @@ def index_field_link(options)
       safe_join(links, ", ")
     end
 
+    # A Blacklight helper_method
+    # @param [Hash] options from blacklight invocation of helper_method
+    # @return the html_safe facet links separated by " > " to indicate facet hierarchy
+
+    def link_to_each_facet_field(options)
+      raise ArgumentError unless options[:config] && options[:value] && options[:config][:helper_facet]
+
+      separator = options.fetch(:config).fetch(:separator, '>')
+      facet_search = options.fetch(:config).fetch(:helper_facet)
+      facet_fields = options[:value].first.split(separator).map(&:strip)
```

Comment:
Wondering about using `options.fetch(:value)` instead. This might mean the above guard clause need not worry about testing if `options[:value]`. Also, it looks as though `options[:value]` assumes an array. I'm not aware of whether it could be a string instead. Would the following work?

```ruby
facet_fields = Array.wrap(options.fetch(:value)).first.split(separator).map(&:strip)
```